Base/usr/share/man/man1/test.md

```diff
@@ -0,0 +1,94 @@
+## Name
+
+test - checks files and compare values
+
+## Synopsis
+
+```**sh
+$ test expression
+$ test
+$ [ expression ]
+$ [ ]
+```
+
+## Description
+
+`test` takes a given expression and sets the exit code according to its truthiness, 0 if true, 1 if false.
+An omitted expression defaults to false, and an unexpected error causes an exit code of 126.
+
+If `test` is invoked as `[`, a trailing `]` is _required_ after the expression, and everything after the `]` is ignored.
```

Comment:
I haven't read the implementation yet, but that's not how it's supposed to work:

```sh
[ -n ] ]
```
This should return 0 because the length of the `]` string is non-zero.

Reply:
Hmmm...so parse away until the end and then check?
it seems `[` is fine with `[ -n ]`, so I'll just make it do that instead.